Kickstarter-iOS/ViewModels/UpdateViewModel.swift

```diff
@@ -5,9 +5,15 @@ import Prelude
 import ReactiveSwift
 import Result
 
+public struct UpdateData {
```

Comment:
this could be private yah? and elsewhere we've called these one-off structs of config data `ConfigData`